python/ray/_private/test_utils.py

```diff
@@ -1394,6 +1394,7 @@ def get_and_run_node_killer(
     lifetime=None,
     no_start=False,
     max_nodes_to_kill=2,
+    node_filter=None,
```

Comment:
nit: since we dont have docstrings for this method, can we name the parameter `node_filter_fn` or similar, to indicate this should be a function?